.scrutinizer.yml

```diff
@@ -4,3 +4,4 @@ imports:
 tools:
     external_code_coverage:
         runs: 5 # No Code Coverage on PHP 5.2 and HHVM
+        timeout: 2700 # 45 minutes
```

Comment:
I normally go for 30 mins. 45 mins can be a long time to wait if someone sends a bad pull request and pushes a few commit to try to fix it, they'll lock up scrutinizer for hours.